### DISCUSSION TITLE
[Project] 첫 번째 프로젝트 기획서

### DISCUSSION THREAD

**Discussion by hemil0102** [hemil0102/XRStudy#30] [3. Project]
# 목표: 
1. 유니티를 활용하여 주변 사물 인식, 추적 및 객체 띄우기 
2. 인식한 객체를 기반으로 ChatGPT 또는 Google API를 통해 이미지 정보 분석하기 
3. 책 페이지를 인식하여 Text 또는 책 내용과 연관된 정보를 띄워본다. 

# 개요: 
1. 유니티를 활용하여 기초적인 3D 객체 구현을 알아본다. 
2. 유니티 + 모바일 환경에서 카메라를 활용하여 증강현실을 구현한다. 
3. 유니티 환경에서 기초적인 통신을 구현한다. 

# 예상 일정: 
~ 6월 18일 기본 사물 인식, 추적 및 객체 띄우기 
~ 7월 2일 네트워크 구현하기 
~ 7월 16일 책 페이지를 인식하여 정보 띄우기 

# 예상 난관: 
1. 유니티 기본 조작 
2. 모바일 카메라 연동하기 
3. 유니티 증강 현실에서 사물 인식하는 법
4. 런타임에서 정보와 객체를 생성하는 법
5. 3D 환경의 수학적 계산 
6. 유니티 API 활용
7. C#
8. 통신 구현 + GPT API Key 경제적 요소

**Comment by jongfeel** [comment]
오 이제 뭔가 대략의 할 일들이 생겼군요.

이 내용에서 목표와 개요를 README.md 파일에 다시 옮겨 적어 보시면 좋을 것 같습니다.
